provider/resources.go

```diff
@@ -561,7 +561,7 @@ func Provider() tfbridge.ProviderInfo {
 					},
 					"metrics_granularity": {
 						Type:     "string",
-						AltTypes: []tokens.Type{awsType(autoscalingMod, "metrics", "MetricsGranularity")},
+						AltTypes: []tokens.Type{awsType(autoscalingMod, "MetricsGranularity", "MetricsGranularity")},
```

Comment:
How is this manifested to the end user (if at all)?

Reply:
It doesn't result in any impactful user-facing changes, just this: https://github.com/pulumi/pulumi-aws/blob/master/examples/lambda-layer-new/index.ts#L17-L18